docs/index.md

```diff
@@ -14,6 +14,14 @@ meta-description: Learn how to install, integrate, configure and develop CKEdito
 <dt>{@linksdk index CKEditor 4 SDK with Editor Samples}</dt><dd>A collection of examples designed to help you get familiar with all editor features.</dd>
 </dl>
 
+## CKEditor 4 Framework Integrations
+
+{@link guide/dev/integration/angular/README CKEditor 4 Angular Integration}
```

Comment:
Maybe some short description under each link, e.g.

> Official CKEditor 4 Angular 2+ integration available through the Angular component. Available as `ckeditor4-angular` npm package.

Reply:
The problem I had with such description is that basically all the integration would have exactly the same description just with different framework name. I'll add it anyway and we will see how it looks like.

Reply:
That's true, quite repeatable. Still it looks a little better than just empty links IMHO.